.env

```
@@ -0,0 +1,21 @@
+PORT=3010
+
+# set the host for the server for outgoing links
+# AFFINE_SERVER_HTTPS=true
+# AFFINE_SERVER_HOST=affine.yourdomain.com
+# or 
+# AFFINE_SERVER_EXTERNAL_URL=https://affine.yourdomain.com
+
+# Folder locations
+DB_DATA_LOCATION=./postgres
+UPLOAD_LOCATION=./storage
+CONFIG_LOCATION=./config
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CONFIG_LOCATION key should go before the DB_DATA_LOCATION key

(UnorderedKey)